pkg/tracker/runner.go

```diff
@@ -53,21 +55,22 @@ func (r *Runner) Start(ctx context.Context, exitCh chan int) error {
 		}()
 		return nil
 	}
-	runnerLog.Info("Created %d trackers", len(trackers))
+	level.Info(logger).Log("msg", fmt.Sprintf("Created %d trackers", len(trackers)))
 
 	var err error
 	masterInstance := ctx.Value(tellorCommon.ContractsTellorContextKey)
 	if masterInstance == nil {
 		contractAddress := common.HexToAddress(cfg.ContractAddress)
 		masterInstance, err = tellor.NewTellor(contractAddress, r.client)
 		if err != nil {
-			runnerLog.Error("Problem creating tellor master instance: %v\n", err)
+			//I believe the dataSerever will handle the os.Exit(1)
```

Comment:
yeah correct here I don't we should log, but just return the message as an error.